src/main/java/net/dries007/tfc/objects/blocks/wood/BlockSupport.java

```diff
@@ -54,6 +55,25 @@
     private static final AxisAlignedBB CONNECTION_W_AABB = new AxisAlignedBB(0.0D, 0.625D, 0.3125D, 0.3125D, 1.0D, 0.6875D);
 
     private static final Map<Tree, BlockSupport> MAP = new HashMap<>();
+    private final Tree wood;
+
+    public BlockSupport(Tree wood)
+    {
+        super(Material.WOOD, Material.WOOD.getMaterialMapColor());
+        if (MAP.put(wood, this) != null) throw new IllegalStateException("There can only be one.");
+        setHardness(2.0F);
+        setHarvestLevel("axe", 0);
+        setSoundType(SoundType.WOOD);
+        this.wood = wood;
+        OreDictionaryHelper.register(this, "support");
+        Blocks.FIRE.setFireInfo(this, 5, 20);
+        setDefaultState(blockState.getBaseState()
+            .withProperty(AXIS, EnumFacing.Axis.Y)
+            .withProperty(NORTH, false)
+            .withProperty(SOUTH, false)
+            .withProperty(EAST, false)
+            .withProperty(WEST, false));
+    }
```

Comment:
IntelliJ looks to have changed this block after I imported the code style and did an automatic code reformat on the file. Should I just leave it?

Reply:
Likely means there's a max line length / hard wrap setting I've set somewhere locally that hasn't made it's way into the code style xml, I'll take a look.

Reply:
I think it moved it up there because it's a constructor and the style wants those as the first things in a class.